test/LetConst/constreassign1.baseline

```diff
@@ -1,8 +1,8 @@
 3
-ReferenceError: Assignment to const
+TypeError: Assignment to const

```

Comment:
Same as above - making pointed changes here and planning for UnitTestFramework switch in a separate PR.